.semaphore/semaphore.yml

```diff
@@ -51,51 +51,151 @@ blocks:
             - PIP_INSTALL_OPTIONS="--user" tools/wheels/build-wheels.sh "${LIBRDKAFKA_VERSION#v}" wheelhouse
             - tar -czf wheelhouse-macOS-${ARCH}.tgz wheelhouse
             - artifact push workflow wheelhouse-macOS-${ARCH}.tgz
-
-  - name: Source package verification with Python 3 (OSX x64) +docs
+  - name: "Wheels: Linux arm64"
+    run:
+      when: "tag =~ '.*'"
     dependencies: []
     task:
       agent:
         machine:
-          type: s1-prod-macos
+          type: s1-prod-ubuntu20-04-arm64-1
       env_vars:
         - name: OS_NAME
-          value: osx
+          value: linux
         - name: ARCH
           value: arm64
       jobs:
         - name: Build
           commands:
-            # use a virtualenv
-            - python3 -m venv _venv && source _venv/bin/activate
-            - pip install -r docs/requirements.txt
-            - pip install -U protobuf
-            # install librdkafka
-            - lib_dir=dest/runtimes/$OS_NAME-$ARCH/native
-            - tools/wheels/install-librdkafka.sh "${LIBRDKAFKA_VERSION#v}" dest
-            - export CFLAGS="$CFLAGS -I${PWD}/dest/build/native/include"
-            - export LDFLAGS="$LDFLAGS -L${PWD}/${lib_dir}"
-            - export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$PWD/$lib_dir"
-            - export DYLD_LIBRARY_PATH="$DYLD_LIBRARY_PATH:$PWD/$lib_dir"
-            # install confluent-kafka
-            - python setup.py build && python setup.py install
-            - make docs
-  - name: "Wheels: Linux arm64"
+            - ./tools/build-manylinux.sh "${LIBRDKAFKA_VERSION#v}"
+            - tar -czf wheelhouse-linux-${ARCH}.tgz wheelhouse
+            - artifact push workflow wheelhouse-linux-${ARCH}.tgz
+  - name: "Wheels: Linux x64"
     run:
       when: "tag =~ '.*'"
     dependencies: []
     task:
       agent:
         machine:
-          type: s1-prod-ubuntu20-04-arm64-1
+          type: s1-prod-ubuntu20-04-amd64-3
       env_vars:
         - name: OS_NAME
           value: linux
         - name: ARCH
-          value: arm64
+          value: x64
       jobs:
         - name: Build
           commands:
             - ./tools/build-manylinux.sh "${LIBRDKAFKA_VERSION#v}"
```

Comment:
We can probably use `tools/wheels/build-wheels.sh` script here for keeping the compatibility with the Travis job.

Reply:
Done